metron-interface/metron-alerts/package.json

```diff
@@ -2,19 +2,18 @@
   "name": "metron-alerts",
   "version": "0.7.1",
   "license": "Apache-2.0",
-  "angular-cli": {},
   "scripts": {
-    "build": "./node_modules/@angular/cli/bin/ng build --prod",
+    "build": "ng build --prod",
     "start": "ng serve --aot",
+    "start:prod": "ng serve --prod",
     "lint": "tslint \"src/**/*.ts\"",
     "test": "ng test --browsers=Chrome",
-    "testCI": "ng test --watch=false --browsers=ChromeHeadless && npm run cypress:ci",
-    "pree2e": "webdriver-manager update",
-    "e2e": "protractor-flake --protractor-path=./node_modules/.bin/protractor --max-attempts=3 -- ./protractor.conf.js",
-    "cypress:run": "cypress run",
+    "test:ci": "ng test --watch=false --browsers=ChromeHeadless",
+    "cypress": "cypress run",
     "cypress:open": "cypress open",
-    "cypress:ci": "ng build --prod && run-p --race start:ci cypress:run",
-    "start:ci": "node ./scripts/alerts-server.js -c alerts_ui_cypress.yml"
```

Comment:
Npm scripts were synchronized across the UI. 
I also removed additional test server impl. and migrated to angular's builtin server to eliminate complexity.